apps/ppm/src/app/routes/private.ts

```diff
@@ -53,6 +54,13 @@ const PrivateRouter: RouterItem[] = [
     component: FeaturesAdvertForm,
     onMenu: false,
   },
+  {
+    path: `/${PrivateRoutesPath.LESSON}${PrivateRoutesPath.GET_BY_ID}`,
+    title: 'Lesson',
+    icon: RouterMenuIcons.MENTOR,
```

Comment:
Why mentor icon on lessons page?

Reply:
I Did not know what else I could to use. So I used this value.

Reply:
if you use     onMenu: false icon should be null